curriculum/challenges/chinese/03-front-end-libraries/front-end-libraries-projects/build-a-25-5-clock.md

```diff
@@ -1,78 +1,78 @@
 ---
 id: bd7158d8c442eddfaeb5bd0f
-title: Build a 25 + 5 Clock
+title: 构建一个番茄时钟
```

Comment:
We specifically changed our project to avoid using "Pomodoro clock" in the English.  This appears to translate to the exactly what we do not want it to state.